config/sample_tpcc_config.xml

```diff
@@ -2,26 +2,26 @@
 <parameters>
 	
     <!-- Connection details -->
-    <dbtype>mysql</dbtype>
-    <driver>com.mysql.jdbc.Driver</driver>
-    <DBUrl>jdbc:mysql://localhost:3306/tpcc</DBUrl>
-    <username>root</username>
+    <dbtype>MONETDB</dbtype>

```

Comment:
Do not change the default sample config file.

Reply:
Create new config xml files named `tpcc_config_monetdb`, `tpch_config_monetdb`, `wikipedia_config_monetdb`.